decoder-js/src/main/java/com/hpe/caf/decoder/PropertyRetriever.java

```diff
@@ -15,16 +15,28 @@
  */
 package com.hpe.caf.decoder;
 
+import com.hpe.caf.system.SystemUtil;
+
 /**
  * Provides access to properties requested during decoding.
  */
 public class PropertyRetriever {
     /**
-     * Gets the value to use for the specified property.
-     * @param key Name of property.
-     * @return Value of property specified.
+     * Gets the value to use for the specified environment variable.
+     * @param key Name of environment variable.
+     * @return Value of environment variable specified.
      */
     public static String getenv(String key){
         return System.getenv(key);
     }
+
+    /**
+     * Gets the value to use for the specified environment variable or property.
+     * @param key Name of environment variable or property.
+     * @param propPrefix The prefix to use when looking up the system property. If null, the key is used as-is.
+     * @return Value of environment variable or property specified.
+     */
+    public static String getenvorprop(String key, String propPrefix){
```

Comment:
Would maybe change this to just `getprop` as discussed here:

https://github.com/WorkerFramework/worker-framework/pull/203/files#r1793533458`